packages/manager/apps/telecom/src/app/telecom/pack/slots/informations/informations.controller.js

```diff
@@ -52,44 +105,19 @@ export default /* @ngInject */ function PackInformationCtrl(
       );
   }
 
-  function init() {
-    self.isLoading = true;
-
-    self.servicesStateLink = {
-      igmap: IG_MAP_URL.FR,
-      travaux: TRAVAUX_URL,
-      guides: GUIDES_URL.FR,
-    };
+  getOrderFollowUp(packName) {
+    const url = `/pack/xdsl/${packName}/orderFollowUp`;
 
-    return $q
-      .all({
-        followUp: getResiliationFollowUp(),
-        cancellable: getIsResiliationCancellable(),
-        associatedLine: getAssociatedLine(),
-      })
-      .then((result) => {
-        self.resiliationFollowUp = result.followUp;
-        self.isCancellable = result.cancellable;
-        self.associatedLine = result.associatedLine;
-        self.isEngaged = moment(
-          $scope.Pack.pack.informations.engagedUpTo,
-        ).isAfter(moment());
-      })
-      .catch((err) => {
-        if (err.status !== 460 && err.status !== 403) {
-          TucToast.error(
-            [
-              $translate.instant('pack_xdsl_oops_an_error_is_occured'),
-              err.data ? err.data.message : '',
-            ].join(' '),
-          );
-        }
-        return $q.reject(err);
-      })
-      .finally(() => {
-        self.isLoading = false;
-      });
+    return this.$http.get(url).then(({ data: orders }) => {
```

Comment:
```suggestion
    return this.$http.get(`/pack/xdsl/${packName}/orderFollowUp`).then(({ data: orders }) => {
```
In addition, if there is a service.js for this feature, you can put the api call in the service